projects/ngx-datatable/src/lib/components/datatable.component.html

```diff
@@ -36,6 +36,7 @@
     [scrollbarH]="scrollbarH"
     [virtualization]="virtualization"
     [loadingIndicator]="loadingIndicator"
+    [spinnerIndicator]="spinnerIndicator"
```

Comment:
I think this is not the right approach. May be an ng-content or template to configure how the loading indicator looks like.

Reply:
Review addressed.
if customLoadingIndicator is set to true then ng-content will be displayed.